interactive-etl/flink-session.yaml

```diff
@@ -11,26 +11,18 @@ spec:
   podTemplate:
```

Comment:
Can it work without the pod template at all? It was added mainly for mounting the inventory file for recommendation app use case.

Reply:
Yes you are right, I just copied it over from the other example so we def don't need it for this one.